sonic_platform_base/sensor_fs.py

```diff
@@ -0,0 +1,159 @@
+"""
+    sensor_fs.py
+
+    File system based sensors implementation. 
+"""
+
+from sonic_platform_base.sensor_base import SensorBase
+from sonic_platform_base.sensor_base import VoltageSensorBase
+from sonic_platform_base.sensor_base import CurrentSensorBase
+import logging
+
+
+class SensorFs(SensorBase):
+    """Implementation of file system based sensor class"""
+
+    __defaults = {
+        'name': '',
+        'sensor': '',
+        'high_thresholds': ['N/A', 'N/A', 'N/A'],
```

Comment:
@bmridul There are 3 high/low threshold values. Can you add comment in code what each value correspond to?

Reply:
ack